app.js

```diff
@@ -1,35 +1,34 @@
 const express = require("express");
 
 const { getCategories } = require("./controllers/categories.controllers");
-const { getReviewById } = require("./controllers/reviews.controllers");
+const {
+  getReviewById,
+  patchReviewById,
+} = require("./controllers/reviews.controllers");
 const { getUsers } = require("./controllers/users.controllers");
+const {
+  handlePSQLErrors,
+  handleCustomErrors,
+  handleInternalErrors,
+  handleInvalidRouteErrors,
+} = require("./controllers/errors.controllers");
 
 const app = express();
 
-app.get("/api/categories", getCategories);
+app.use(express.json());
```

Comment:
I'm just really happy that you've put this in now, when you need it, rather than right at the start of the project before it's required.